templates/manila/config/00-config.conf

```diff
@@ -25,13 +25,15 @@ db_max_retries = -1
 [keystone_authtoken]
 www_authenticate_uri={{ .KeystonePublicURL }}
 auth_url = {{ .KeystoneInternalURL }}
+memcached_servers = {{ .MemcachedServersWithInet }}
 auth_type = password
 project_domain_name = Default
 user_domain_name = Default
 project_name = service
 username = {{ .ServiceUser }}
 password = {{ .ServicePassword }}
 interface = internal
+token_cache_time = 600
```

Comment:
WIP: This is just to test whether tempest tests pass when the token timeout is increased.